diff -ur rpm-4.2/python/Makefile.in rpm-4.2.new/python/Makefile.in
--- rpm-4.2/python/Makefile.in	2003-03-10 11:09:42.000000000 -0800
+++ rpm-4.2.new/python/Makefile.in	2004-10-23 18:16:25.802568264 -0700
@@ -209,7 +209,8 @@
 
 PYVER = @WITH_PYTHON_VERSION@
 
-pylibdir = $(shell python -c 'import sys; print sys.path[1]')
+
+pythondir = $(shell python -c 'from distutils import sysconfig; print sysconfig.get_python_lib(1,0)')
 pyincdir = $(prefix)/include/python${PYVER}
 
 SUBDIRS = rpmdb test
@@ -242,10 +243,9 @@
 
 LDADD = 
 
-pythondir = $(pylibdir)/site-packages
 python_PROGRAMS = rpmmodule.so
 
-rpmdbdir = $(pylibdir)/site-packages/rpmdb
+rpmdbdir = $(pythondir)/rpmdb
 rpmdb_PROGRAMS = _rpmdb.so
 
 noinst_PROGRAMS = poptmodule.so
Only in rpm-4.2.new/python: Makefile.in.orig
diff -ur rpm-4.2/python/rpmdb/Makefile.in rpm-4.2.new/python/rpmdb/Makefile.in
--- rpm-4.2/python/rpmdb/Makefile.in	2003-03-10 11:09:42.000000000 -0800
+++ rpm-4.2.new/python/rpmdb/Makefile.in	2004-10-23 18:16:25.803568112 -0700
@@ -207,13 +207,13 @@
 
 PYVER = @WITH_PYTHON_VERSION@
 
-pylibdir = $(shell python -c 'import sys; print sys.path[1]')
+pythondir = $(shell python -c 'from distutils import sysconfig; print sysconfig.get_python_lib(1,0)')
 
 EXTRA_DIST = \
 	__init__.py dbobj.py db.py dbrecio.py dbshelve.py dbtables.py dbutils.py
 
 
-rpmdbdir = $(pylibdir)/site-packages/rpmdb
+rpmdbdir = $(pythondir)/rpmdb
 rpmdb_SCRIPTS = \
 	__init__.py dbobj.py db.py dbrecio.py dbshelve.py dbtables.py dbutils.py
 
Only in rpm-4.2.new/python/rpmdb: Makefile.in.orig
